Modules/SubModule_Import/xASL_bids_BIDSifyCheckTimeEncoded.m

```diff
@@ -0,0 +1,90 @@
+function [jsonIn,jsonOut,bTimeEncoded,bTimeEncodedFME] = xASL_bids_BIDSifyCheckTimeEncoded(jsonIn,jsonOut)
+%xASL_bids_BIDSifyCheckTimeEncoded Check for time encoded sequence
+%
+% FORMAT: [jsonIn,jsonOut] = xASL_bids_BIDSifyCheckTimeEncoded(jsonIn,jsonOut)
+%
+% INPUT:
+%   jsonIn    - JSON with the input fields - from DICOMs (REQUIRED)
+%   jsonOut   - ordered and checked JSON structure
```

Comment:
Looks like copy paste, but we have to be more specific here, because in this function, it is pretty unclear why jsonOut is on input :)

"jsonOut - Output JSON in progress from the parent function xASL_bids_BIDSifyASLJSON (REQUIRED)"

Reply:
Easy fix 
4c6babf 👍